src/appellate/appellate.js

```diff
@@ -67,6 +69,153 @@ AppellateDelegate.prototype.dispatchPageHandler = function () {
   }
 };
 
+AppellateDelegate.prototype.handleAcmsAttachmentPage = async function () {
+  const processAttachmentPage = async () => {
+    let caseSummary = JSON.parse(sessionStorage.caseSummary);
+    this.pacer_case_id = caseSummary.caseDetails.caseId;
+
+    const options = await getItemsFromStorage('options');
+    if (options['recap_enabled']) {
+      let vueData = JSON.parse(sessionStorage.recapVueData);
+      let requestBody = {
+        caseDetails: caseSummary.caseDetails,
+        docketEntry: vueData.docketEntry,
+        docketEntryDocuments: vueData.docketEntryDocuments,
+      };
+      this.recap.uploadDocket(
+        this.court,
+        this.pacer_case_id,
+        JSON.stringify(requestBody),
+        'ACMS_ATTACHMENT_PAGE',
+        (ok) => {
+          console.log('in cb');
+          if (ok) {
+            console.log('cb success');
```

Comment:
Some stray logging here. Can you please remove?

Reply:
Sure!